src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -222,13 +219,10 @@ export default defineComponent({
       return 'total';
     });
 
-    const totalAmount = computed(() => getTxAmountTotal.value(
-      txWrapped.value,
-      getTxDirection.value(props.transaction),
-    ));
+    const totalAmount = computed(() => getTxAmountTotal.value(txWrapped.value, direction.value));
 
     const singleToken = computed((): ITokenResolved => ({
-      isReceived: getTxDirection.value(props.transaction) === TX_FUNCTIONS.received,
+      isReceived: direction.value === TX_FUNCTIONS.received,
```

Comment:
Side note: isn't this a mistake that the direction values are stored in the `TX_FUNCTION` collection?

Reply:
Looks like a mistake to me.